nginx-shibboleth/install.sls

```diff
@@ -1,10 +1,68 @@
 {% from "nginx-shibboleth/map.jinja" import nginx_shibboleth with context %}
 
 include:
+  - nginx.ng
   - .service
 
-nginx-shibboleth:
+{% if salt.grains.get('os_family') == 'RedHat' %}
+add_shibboleth_yum_repo:
+  pkgrepo.managed:
+    - name: security_shibboleth
+    - humanname: Shibboleth (CentOS_7)
+    - type: rpm-md
+    - baseurl: http://download.opensuse.org/repositories/security:/shibboleth/CentOS_7/
+    - gpgcheck: True
+    - gpgkey: http://download.opensuse.org/repositories/security:/shibboleth/CentOS_7/repodata/repomd.xml.key
+    - enabled: True
+    - require_in:
+        - pkg: install_service_provider
+{% endif %}
+
+install_service_provider:
   pkg.installed:
     - pkgs: {{ nginx_shibboleth.pkgs }}
```

Comment:
`nginx_shibboleth.pkgs` should be `nginx_shibboleth.pkg` or the key in `map.jinja` needs to be changed